liquid_node/configuration.py

```diff
@@ -1,43 +1,31 @@
+import configparser
 import time
 from collections import OrderedDict
-import configparser
+from distutils.util import strtobool
 from pathlib import Path
-from .util import import_string
-from .jobs import Job
 
-from liquid_node.jobs import liquid
-from liquid_node.jobs import hoover
-from liquid_node.jobs import dokuwiki
-from liquid_node.jobs import rocketchat
-from liquid_node.jobs import nextcloud
-from liquid_node.jobs import ci
+from .util import import_string
+from liquid_node.jobs import ci, Job
 
 
 class Configuration:
+    _core_apps = ('liquid', 'hoover', 'dokuwiki', 'rocketchat', 'nextcloud',)
+    _allways_on_apps = ('liquid', 'hoover',)
```

Comment:
Typo: `always`.

I think we should name the first list `ALL_APPS` and the second one `CORE_APPS`, both in the global namespace. The `core` is something we can't do without, so a comment `# These apps can't be turned off.` should be enough to clarify this.

I guess we should be able to disable running `hoover` too, so please open an issue to track that.